packages/atomic/src/components/common/smart-snippets/atomic-smart-snippet-suggestions/smart-snippet-suggestions-common.tsx

```diff
@@ -143,7 +144,9 @@ export class SmartSnippetSuggestionCommon {
                 .getQuestionsList()
                 .cancelPendingSelectSource(relatedQuestion.questionAnswerId)
             }
-          ></atomic-smart-snippet-source>
+          >
+            <slot name="attributes" slot="source-attributes"></slot>
```

Comment:
I now realize that in the example I gave you, it looks like I reversed this.

I meant for the slot in `atomic-smart-snippet` to be named "source-attributes" and the one in `atomic-smart-snippet-source` to be named "attributes".

So this:
```suggestion
            <slot name="source-attributes" slot="attributes"></slot>
```

Reply:
Changed!